README.md

```diff
@@ -62,6 +62,15 @@ Inside the `l12n-shell`, you can use the following commands:
   - you can also run pre-configured queries using the examples. Run `l12n -h` to
     see the list of examples.
 
+## About the stack
+
+Infrastructure is managed by Terraform.
+
+We use Terragrunt to:
+
+- DRY the Terraform config
```

Comment:
❕ I don't think DRY is sufficiently well known to have it in a Readme (ok for commits).  The clearer the better

Reply:
added a link to wikipedia